pkg/agent/agent_windows.go

```diff
@@ -218,6 +287,79 @@ func (i *Initializer) prepareOVSBridgeOnHNSNetwork() error {
 	return nil
 }
 
+func (i *Initializer) prepareOVSConfigForVM() error {
+	klog.InfoS("Performing OVS configuration", "hostIfName", i.nodeConfig.UplinkNetConfig.Name)
+	hostIfName := i.nodeConfig.UplinkNetConfig.Name
+	uplinkIfName := util.GenerateUplinkInterfaceName(hostIfName)
+	ovsPorts, ovsErr := i.ovsBridgeClient.GetPortList()
+	if ovsErr != nil {
+		klog.Errorf("Failed to list OVS ports")
+		return ovsErr
+	}
+	for index := range ovsPorts {
+		port := &ovsPorts[index]
+		if port.Name == hostIfName {
+			klog.Info("Uplink and host configuration exists in OVS")
+			return nil
+		}
+	}
+
+	success := false
+	uplinkExternalIDs := map[string]interface{}{
+		interfacestore.AntreaInterfaceTypeKey: interfacestore.AntreaUplink,
+	}
+	// TODO: Have a separate function for creation of pair ports
+	// Create uplink port on OVS.
+	uplinkUUID, ovsErr := i.ovsBridgeClient.CreatePort(uplinkIfName, uplinkIfName, uplinkExternalIDs)
+	if ovsErr != nil {
+		return fmt.Errorf("failed to create uplink port on OVS for %s: %v", uplinkIfName, ovsErr)
+	}
+
+	// Manual clean up of OVS configurations is required, when agent exits abruptly.
```

Comment:
"or the cleanup fails"?

Reply:
Did not get what you mean here?
// Manual clean up of OVS configurations is required, when agent exits abruptly.

Reply:
I meant should we add "cleanup fails" as another case that requires "manual cleanup", like:

> // Manual cleanup of OVS configurations is required, when agent exits abruptly, or when the auto cleanup operation fails.".

Reply:
Thanks for clarification. Updated it.